primefaces/src/main/resources/META-INF/resources/primefaces/core/core.widget.js

```diff
@@ -587,6 +581,25 @@ if (!PrimeFaces.widget) {
             }
             
             return this.cfg.formId;
+        },
+
+        /**
+         * Registers a DOM element such that its removal from the DOM results in the widget becoming "detached,"
+         * subsequently leading to its removal upon completion of the AJAX call.
+         *
+         * @param {JQuery | HTMLElement} watchElement The HTML element that if removed from the DOM will detach this widget.
+         * @since 14.0.0
+         */
+        bindDomRemovalEvent: function(watchElement) {
```

Comment:
+1 in general for this PR
i think we should just align the method name a bit - its not clear the widget will be destroyed when the passed dom element is removed

Reply:
Yep I didn't love the name either.  How about `registerDestroyListener`, or `registerRemovalWatcher`?  I am open to any names...

Reply:
we already have registerDestroyListener

maybe something like
bindLifetimeOnElement
destroyOnElementRemoval

Reply:
i like `destroyOnElementRemoval`